tests/tasks/snowflake/test_snowflake.py

```diff
@@ -76,3 +76,46 @@ def test_execute_fetchall(self, monkeypatch):
         ).run()
 
         assert output == ["TESTDB"]
+
+
+class TestSnowflakeQueriesFromFile:
```

Comment:
There should be a test that covers the functionality i.e. that the correct calls are made to the connection and that mocked return values are handled correctly